openstates/scrape/jurisdiction.py

```diff
@@ -44,7 +45,12 @@ def __init__(self):
 
     @property
     def classification(self):
-        return "state" if self.name != "United States" else "country"
+        if self.name == "United States":
```

Comment:
`if any(c == self.name for c in ["United States", "South Africa"])`?

Reply:
From there, we could just have a `COUNTRIES` key or something we could filter on.